contracts/yieldStrategy/gmx/GMXBatchingManager.sol

```diff
@@ -21,23 +21,28 @@ contract GMXBatchingManager is IGMXBatchingManager, OwnableUpgradeable, Pausable
     using FullMath for uint128;
     using SafeCast for uint256;
 
-    mapping(address => UserDeposit) public userDeposits;
-    mapping(uint256 => RoundDeposit) public roundDeposits;
-
-    modifier onlyKeeper() {
-        if (_msgSender() != keeper) revert CallerNotKeeper();
-        _;
+    struct VaultBatchingState {
+        uint256 currentRound;
+        uint256 roundGlpBalance;
+        mapping(address => UserDeposit) userDeposits;
+        mapping(uint256 => RoundDeposit) roundDeposits;
     }
-    uint256 public currentRound;
-    uint256 public roundGlpBalance;
+
+    IERC4626[] public vaults;
+    mapping(IERC4626 => VaultBatchingState) public vaultBatchingState;
+    IERC4626 public defaultVault; // used for depositing harvested rewards
```

Comment:
defaultVault is never set

Reply:
TODO
1. remove defaultVault
2. add another function which does the first if statement in depositToken